spec/models/spot_spec.rb

```diff
@@ -33,5 +33,14 @@
   describe 'associations' do
     it { should belong_to(:user) }
     it { should belong_to(:topic) }
+
+    context 'when exceeding 10th spot creation limit' do
+      let(:user)    { create(:user) }
+      let!(:spots)  { create_list(:spot, 10, user_id: user.id) }
+      it 'does not create the spot' do
+        expect { create(:spot, user_id: user.id) }
+          .to raise_error(ActiveRecord::RecordInvalid)
```

Comment:
You should check if the record is valid and the errors instead of checking the exception here